openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/myData.spec.js

```diff
@@ -159,7 +146,7 @@ describe('MyData page should work', () => {
     });
   });
 
-  it('Recent view section and redirection should work for table entity', () => {
+  it.only('Recent view section and redirection should work for table entity', () => {
```

Comment:
Any reason why we want to run only this test?

Reply:
Oh shit, Thanks for highlighting! 🙏🏻

Reply:
push the changes